UnityTestingTool/docs/README-UnityTestingTool

```diff
@@ -0,0 +1,91 @@
+Unity Testing Tool (Mar. 23, 2019)
```

Comment:
Let's use the Asciidoc format. Team guide: https://github.com/UCSolarCarTeam/Development-Resources/tree/master/Development-Guide/AsciiDoc-Guide